src/aura/AuraAvatarMultiToken.sol

```diff
@@ -429,49 +393,46 @@ contract AuraAvatarTwoToken is BaseAvatar, PausableUpgradeable, AuraAvatarUtils,
 
     /// @notice Takes a given amount of assets from the owner and stakes them on the AURA Booster. Can only be called by owner.
     /// @dev This also initializes the lastClaimTimestamp variable if there are no other deposits.
-    /// @param _amountAsset1 Amount of asset1 to be staked.
-    /// @param _amountAsset2 Amount of asset2 to be staked.
-    function deposit(uint256 _amountAsset1, uint256 _amountAsset2) external onlyOwner {
-        if (_amountAsset1 == 0 && _amountAsset2 == 0) {
-            revert NothingToDeposit();
-        }
-
-        uint256 bptDeposited1 = baseRewardPool1.balanceOf(address(this));
-        uint256 bptDeposited2 = baseRewardPool2.balanceOf(address(this));
-        if (bptDeposited1 == 0 && bptDeposited2 == 0) {
+    /// @param _pids Pids target to stake into
+    /// @param _amountAssets Amount of assets to be staked.
+    function deposit(uint256[] memory _pids, uint256[] memory _amountAssets, bool initialDeposit) external onlyOwner {
```

Comment:
Should we check if `_pids` are a subset of the pids in storage?

Reply:
checked on deposit and included an error for its case `PidNotIncluded(uint256 pid)` in 1bf3520289ab8590692162543359a3039d54602b